src/execution.jl

```diff
@@ -704,7 +708,6 @@ macro bprofile(args...)
     return esc(
         quote
             local $tmp = $BenchmarkTools.@benchmarkable $(args...)
-            $BenchmarkTools.warmup($tmp)
             $(hasevals(params) ? :() : :($BenchmarkTools.tune!($tmp)))
             $BenchmarkTools.Profile.clear()
             $BenchmarkTools.@profile $BenchmarkTools.run($tmp)
```

Comment:
If `evals` is set, we don't run a warmup before profiling.

Reply:
This PR adds the warmup to the `run` method, so it does get warmed up, but I see what you mean that the @profile now has no way to only measure the warm version.